templates/weave.go

```diff
@@ -1,13 +1,7 @@
-package network
+package templates
 
-import "github.com/rancher/rke/services"
-
-func GetWeaveManifest(weaveConfig map[string]string) string {
-	rbacConfig := ""
-	if weaveConfig[RBACConfig] == services.RBACAuthorizationMode {
-		rbacConfig = getWeaveRBACManifest()
-	}
-	return `
+const WeaveTemplate = `
```

Comment:
Weave template doesn't contain if condition for RBAC

Reply:
Fixed.